apps/blog/src/posts/state-of-pwa-1/state-of-pwa-1.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: State of PWA - November 2023
+excerpt: See what's going on in the world of Progressive Web Apps!
+description: This article covers what's new for Progressive Web Apps over the last year or so - from numbers on adoption to new APIs and more.
+date: 2023-11-22
+updatedDate: 2023-11-22
+trending: true
+featured: true
+image: 
```

Comment:
We are going to need an image. Can you work with Sana if you just need a generic one?